### PR TITLE
[Release] sdk/resourcemanager/postgresql/armpostgresqlflexibleservers/3.0.0

### PR DESCRIPTION
**Pull request by Alancere**
https://github.com/Azure/sdk-release-request/issues/4001

### CODE REVIEW

**Comment by Alancere** [comment]
The breaking changes are introduced and approved in [this PR](https://github.com/Azure/azure-rest-api-specs/pull/23294).

File: sdk/resourcemanager/postgresql/armpostgresqlflexibleservers/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 3.0.0 (2023-04-28)
+### Breaking Changes
+
+- Const `ArmServerKeyTypeSystemAssigned` from type alias `ArmServerKeyType` has been removed
```
Comment (tadelesh) [inline sdk/resourcemanager/postgresql/armpostgresqlflexibleservers/CHANGELOG.md:6]:
```suggestion
- Const `ArmServerKeyTypeSystemAssigned` from enum `ArmServerKeyType` has been removed
```

File: sdk/resourcemanager/postgresql/armpostgresqlflexibleservers/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+- Const `ReplicationRoleGeoSyncReplica`, `ReplicationRoleSecondary`, `ReplicationRoleSyncReplica`, `ReplicationRoleWalReplica` from type alias `ReplicationRole` has been removed
```
Comment (tadelesh) [inline sdk/resourcemanager/postgresql/armpostgresqlflexibleservers/CHANGELOG.md:7]:
```suggestion
- Const `ReplicationRoleGeoSyncReplica`, `ReplicationRoleSecondary`, `ReplicationRoleSyncReplica`, `ReplicationRoleWalReplica` from enum `ReplicationRole` has been removed
```

**Comment by tadelesh** [comment]
@JeffreyRichter could you help to review and approve this major version release?

**Comment by tadelesh** [comment]
> What are you asking me to do here? If the service broke, then this package must break too and that requires a major version update. Is there anything for me to think about or just want me to add some label?

Since Go major version will have more impact to the customer, Rick is asking that we need get approval from you for each major version release. For each release with breaking change, we will attach the service swagger breaking PR which always already get approval from breaking change review board. If there is no further concern, you could just add a label Approved Major Version, then we can continue with the release. Thank you so much.

**Review by tadelesh — APPROVED** [review]

**Pull request merged**